src/Dev/Tasks/FileMigrationHelper.php

```diff
@@ -93,26 +90,21 @@ public function run($base = null)
             $base = PUBLIC_PATH;
         }
 
-        // Set max time and memory limit
-        Environment::increaseTimeLimitTo();
-        Environment::increaseMemoryLimitTo();
-
-        $this->logger->info('MIGRATING SILVERSTRIPE 3 LEGACY FILES');
+        $this->logger->notice('Step 1/2: Migrate 3.x legacy files to new format');
         $ss3Count = $this->ss3Migration($base);
 
-        $this->logger->info('NORMALISE SILVERSTRIPE 4 FILES');
+        $this->logger->notice('Step 2/2: Migrate <4.4 files to new format');
+
         $ss4Count = 0;
         if (class_exists(Versioned::class) && File::has_extension(Versioned::class)) {
             Versioned::prepopulate_versionnumber_cache(File::class, Versioned::LIVE);
             Versioned::prepopulate_versionnumber_cache(File::class, Versioned::DRAFT);
 
-            $this->logger->info('Looking at live files');
```

Comment:
Not sure why this is omitted, it could be useful.

Reply:
I found it a bit weird to have those two messages when no files needed migrating. I've made it a bit clearer now what the outcome of each of those steps is, and re-introduced the messages. You're right as in there's some value in flagging that both live/draft are looked at separately, otherwise people might get confused why files show up twice in the logs. Made this a bit more verbose and reintroduced those messages.